src/docs/background/how-to-integrate-3rd-party-payment-gateway/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: How to integrate a 3rd-party payment gateway with your marketplace
+slug: how-to-integrate-3rd-party-payment-gateway
+updated: 2021-03-19
+category: background
+ingress:
+  This guide describes, on a high-level, how to integrate any 3rd-party payment gateway with your Flex-powered marketplace.
+published: true
+---
+
+Sharetribe Flex provides out-of-the-box integration with Stripe. To reach markets not supported by Stripe, relying on another payment gateway can be crucial. This guide describes on a high-level, without going into the details of specific payment gateways, how to integrate any 3rd-party payment gateway (such as [PayPal Commerce Platform](https://www.paypal.com/us/business/platforms-and-marketplaces), [MANGOPAY](https://www.mangopay.com/marketplaces/), or [Adyen for Platforms](https://www.adyen.com/our-solution/marketplaces-and-platforms)) with Sharetribe Flex.
+
+## Prerequisites
+
+Before reading this guide, you should be familiar with the following Sharetribe Flex features:
+
+* [Privileged transitions](https://www.sharetribe.com/docs/background/privileged-transitions/)
+* [Events](https://www.sharetribe.com/docs/references/events/)
+* [Reacting to events](https://www.sharetribe.com/docs/cookbook-events/reacting-to-events/)
+* [Extended data](https://www.sharetribe.com/docs/references/extended-data/)
```

Comment:
If you remove the `https://www.sharetribe.com/docs ` prefix from the URLs, they also work locally and in deploy previews.